api/src/main/java/org/openmrs/api/context/Daemon.java

```diff
@@ -102,7 +108,68 @@ public void run() {
 
 		return (Module) startModuleThread.returnedObject;
 	}
-	
+
+	/**
+	 * This method should not be called directly, only {@link ContextDAO#createUser(User, String)} can
+	 * legally invoke {@link #createUser(User, String)}.
+	 * 
+	 * @param user
```

Comment:
Did you just forget to document these params?

Reply:
Yes, I'll do that in the next commit.